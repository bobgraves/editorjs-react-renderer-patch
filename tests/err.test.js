import ERR from '../index.jsx';

describe('ERR renderer:', () => {
  const err = shallow(<ERR />);

  it('should match snapshot', () => expect(err).toMatchSnapshot());
});
