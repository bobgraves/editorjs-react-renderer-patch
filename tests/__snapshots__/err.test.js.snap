// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ERR renderer: should match snapshot 1`] = `""`;
